src/console_reporter.cc

```diff
@@ -56,7 +56,7 @@ bool ConsoleReporter::ReportContext(const Context& context) {
 BENCHMARK_EXPORT
 void ConsoleReporter::PrintHeader(const Run& run) {
   std::string str =
-      FormatString("%-*s %13s %15s %12s", static_cast<int>(name_field_width_),
+      FormatString("%-*s %13s %15s %15s", static_cast<int>(name_field_width_),
```

Comment:
My main concern would be that the lines become even longer now.
I suppose i'd be fine with the limit bump, but degradation of the output makes me do a second take.